### ISSUE TITLE
matter integration stopped working yesterday

### ISSUE THREAD

**Issue by rchiileea** [home-assistant-libs/python-matter-server#253]
yesterday matter stopped working, the server is still running but the integration is failing with matter client not ready 
![image](https://user-images.githubusercontent.com/87599807/229720897-053991a0-11b7-4ef6-b42a-b718a5ff8a52.png)

this is the log

``` Unexpected exception: 0 is not a valid AccessControl.Enums.Privilege
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/matter/__init__.py", line 127, in _client_listen
    await matter_client.start_listening(init_ready)
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/client.py", line 276, in start_listening
    nodes = [
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/client.py", line 277, in <listcomp>
    MatterNode(dataclass_from_dict(MatterNodeData, x))
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/models/node.py", line 212, in __init__
    self.update(node_data)
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/models/node.py", line 299, in update
    self.endpoints[endpoint_id] = MatterEndpoint(
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/models/node.py", line 57, in __init__
    self.update(attributes_data)
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/models/node.py", line 186, in update
    self.set_attribute_value(attribute_path, attribute_value)
  File "/usr/local/lib/python3.10/site-packages/matter_server/client/models/node.py", line 177, in set_attribute_value
    attribute_value = parse_value(
  File "/usr/local/lib/python3.10/site-packages/matter_server/common/helpers/util.py", line 116, in parse_value
    return [
  File "/usr/local/lib/python3.10/site-packages/matter_server/common/helpers/util.py", line 117, in <listcomp>
    parse_value(name, subvalue, get_args(value_type)[0])
  File "/usr/local/lib/python3.10/site-packages/matter_server/common/helpers/util.py", line 113, in parse_value
    return dataclass_from_dict(value_type, value)
  File "/usr/local/lib/python3.10/site-packages/matter_server/common/helpers/util.py", line 214, in dataclass_from_dict
    **{
  File "/usr/local/lib/python3.10/site-packages/matter_server/common/helpers/util.py", line 215, in <dictcomp>
    field.name: parse_value(
  File "/usr/local/lib/python3.10/site-packages/matter_server/common/helpers/util.py", line 164, in parse_value
    return value_type(value)
  File "/usr/local/lib/python3.10/enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "/usr/local/lib/python3.10/enum.py", line 710, in __new__
    raise ve_exc
ValueError: 0 is not a valid AccessControl.Enums.Privilege```

have tried powering system down and removing the dongle and powering up and reconnecting it, as well as restoring from a backup.

**Comment by karlshea** [comment]
I was getting that error on 3.2.0 until upgrading to Home Assistant 2023.4.0, then it started working again.

**Comment by rchiileea** [comment]
issue is resolved with latest 2023.4.0
